clouddrift/analysis.py

```diff
@@ -1159,3 +1169,219 @@ def unpack_ragged(
         rows = [rows]
 
     return [ragged_array[indices[n] : indices[n + 1]] for n in rows]
+
+
+def extract_inertial_from_position(
+    longitude: np.ndarray,
+    latitude: np.ndarray,
+    p: int,
```

Comment:
You already know what I'm going to say about this `p` here.

Needs a descriptive name. Docstring says duration of wavelet (so, some kind of time?) but then it says it must be one or greater and it's an integer. Is it some kind of order or number of wavelets to use?

Reply:
Sure, I can change that one.